lib/active_model/serializer.rb

```diff
@@ -105,7 +105,7 @@ def self.root_name
       name.demodulize.underscore.sub(/_serializer$/, '') if name
     end
 
-    attr_accessor :object, :root, :meta, :meta_key, :scope
+    attr_accessor :object, :root, :meta, :meta_key, :links, :scope
```

Comment:
Maybe it should behave like `include_meta` such that there is an `include_links` (which calls `include_pagination`)?  I've written some more (please don't shoot me :) thinking about how to 'pass pagination' in through the 'links' adapter option in https://github.com/rails-api/active_model_serializers/pull/1041/files#r36998222